src/scripts/delete-user.ts

```diff
@@ -9,7 +9,7 @@ import * as O from 'fp-ts/lib/Option';
 import { flow, pipe } from 'fp-ts/lib/function';
 import * as Console from 'fp-ts/lib/Console';
 
-const deleteUser = (
+export const deleteUser = (
```

Comment:
> Write doc now that this is exported.

What do you mean by this comment?

Reply:
exported (aka public) members should have typedoc.

Reply:
So you mean have some sort of d.ts document. Where should I put it (i.e. which directory), what is an appropriate name to call it, and should it include all of the exports across the scripts section of the oktagon project?

Reply:
Typedocs added